go.mod

```diff
@@ -3,72 +3,75 @@ module github.com/medibloc/panacea-core/v2
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.24.0
-	github.com/btcsuite/btcd v0.22.1
+	github.com/CosmWasm/wasmd v0.30.0
```

Comment:
For IBC v4, `wasmd` must be also upgraded to v0.30.0 that is the first version based on IBC v4.
https://github.com/CosmWasm/wasmd/blob/main/CHANGELOG.md#v0300-2022-12-02